lib/interface.js

```diff
@@ -7,6 +7,9 @@ module.exports = function (Target) {
   Target.prototype.toQuery = function (tz) {
     let data = this.toSQL(this._method, tz);
     if (!Array.isArray(data)) data = [data];
+    if (!data.length) {
```

Comment:
Result of breakage was introduction of `reduce` after .map - `reduce` throws an error on empty array when there is no default collector.